crates/ruff_linter/src/codes.rs

```diff
@@ -1060,6 +1060,7 @@ pub fn code_to_rule(linter: Linter, code: &str) -> Option<(RuleGroup, Rule)> {
         (Ruff, "064") => rules::ruff::rules::NonOctalPermissions,
         (Ruff, "065") => rules::ruff::rules::LoggingEagerConversion,
         (Ruff, "066") => rules::ruff::rules::PropertyWithoutReturn,
+        (Ruff, "070") => rules::ruff::rules::NonEmptyInitModule,
```

Comment:
Nit: I prefer avoiding "holes" and instead assign rule ids on the rule that comes first